pyclam/criterion.py

```diff
@@ -176,6 +227,29 @@ def _predict_auc(self, manifold: Manifold) -> Dict[Cluster, float]:
         return predicted_auc
 
 
+class RegressionTreePaths(SelectionCriterion):
+    """ Uses the decision paths of the Regression Tree meta-ml models to select best clusters for graph.
+    """
+    def __init__(self, predict_auc: Callable[[Cluster], float], *, mode: str = 'ranked'):
```

Comment:
should include modes in docstring somewhere

Reply:
fixed